functional_tests/testdata/dotnet/README.md

```diff
@@ -13,3 +14,25 @@ Running this container inside a Kubernetes cluster under observation of the oper
 ## Develop
 
 Login to quay.io and push with `make push`
+Make sure for new image repositories you make the repository public
+- Arm based machines can have issues running dockerx build commands with this test image due to QEMU and .NET support, see: https://github.com/dotnet/dotnet-docker/issues/3832
+- On an M2 Mac running Docker Desktop 25.0.0, was able to get the dockerx build to run after setting `docker buildx create --use --name multi-arch-builder`
+
+### Debug Env Vars
+
+```
+env:
+  - name: OTEL_LOG_LEVEL
+    value: DEBUG
+  - name: OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED
+    value: "true"
+  # This is needed to bypass an error thrown by auto-instrumentation:
```

Comment:
Perhaps the comment should be under the name of the corresponding env var.

Reply:
Split up these notes. Should be good now.